### ISSUE TITLE
space-after-colon incorrectly warns in @at-root :pseudo selector

### ISSUE THREAD

**Issue by stowball** [sasstools/sass-lint#1020]
What version of Sass Lint are you using?

1.10.2

Please include any relevant parts of your configuration

`space-after-selector: 1`

What did you do? Please include the actual source code causing the issue.

```scss
.foo {
    @at-root button#{&}:hover {
    }
}
```

What did you expect to happen?

No errors

What actually happened? Please include any error messages given to you by Sass Lint.

The `@at-root` line complains that "Space expected after `:`" in `:hover`

If you're using a IDE plugin have you tried the CLI too?

Yes

**Comment by DanPurdy** [comment]
Released in 1.12.0
